daemon/execdriver/lxc/driver.go

```diff
@@ -773,15 +787,17 @@ func (d *driver) generateEnvConfig(c *execdriver.Command) error {
 }
 
 // Clean not implemented for lxc
-func (d *driver) Clean(id string) error {
+func (d *Driver) Clean(id string) error {
 	return nil
 }
 
+// TtyConsole is for when using a container interactively
```

Comment:
It stores the read and write ends of the containers's pty pipe.

Not sure if that makes sense?